### ISSUE TITLE
Should be able to play songs directly from spotify

### ISSUE THREAD

**Issue by robocoder99** [forslund/mycroft-media-skills#12]
Perhaps a "play song [name] from spotify" command? I just think this would be extremely useful as it already exists for the other platforms such as local music.

**Comment by forslund** [comment]
I agree that the spotify skill is in need of some love. Playing single songs is a bit tricky since there usually exists multiple versions and it's hard to determine exactly the one the user wants to hear (unless it's only songs in the users playlists).

I'm doing a refactoring to make the skills more similar and I will see if I can cook up some way to search better.

play intergalactic

```
play intergalactic from spotify
play intergalactic by beastie boys from spotify
play intergalactic from album hello nasty from spotify
play intergalactic from album hello nasty by beastie boys from spotify
```
